packages/fiori/src/FlexibleColumnLayout.ts

```diff
@@ -0,0 +1,821 @@
+	*  - <code>endArrowContainerRole</code>: the accessibility role for the second arrow container (between the <code>mid</code> and <code>end</code> columns)
+	*  - <code>endColumnRole</code>: the accessibility role for the <code>endColumn</code>
+	*
+	* @type {object}
+	* @public
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.accessibilityRoles
+	* @since 1.1.0
+	*/
+	@property({ type: Object })
+	accessibilityRoles!: AccessibilityRoles;
+
+	/**
+	* Defines the component width in px.
+	*
+	* @type {sap.ui.webc.base.types.Float}
+	* @defaultvalue 0
+	* @private
+	*/
+	@property({ validator: Float, defaultValue: 0 })
+	_width!: number;
+
+	/**
+	* Defines the effective columns layout,
+	* based on both the <code>layout</code> property and the screen size.
+	* Example: [67%, 33%, 0], [25%, 50%, 25%], etc.
+	*
+	* @type {object}
+	* @defaultvalue undefined
+	* @private
+	*/
+	@property({ type: Object, defaultValue: undefined })
+	_columnLayout?: ColumnLayout;
+
+	/**
+	* Defines the visible columns count - 1, 2 or 3.
+	*
+	* @type {sap.ui.webc.base.types.Integer}
+	* @defaultvalue 1
+	* @private
+	*/
+	@property({ validator: Integer, defaultValue: 0 })
+	_visibleColumns!: number;
+
+	/**
+	* Allows the user to replace the whole layouts configuration
+	*
+	* @type {object}
+	* @private
+	* @sap-restricted
+	*/
+	@property({ type: Object, defaultValue: undefined })
+	_layoutsConfiguration?: LayoutConfiguration;
+
+	/**
+	* Defines the content in the start column.
+	* @type {HTMLElement}
+	* @slot
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.startColumn
+	* @public
+	*/
+	@slot()
+	startColumn!: Array<HTMLElement>;
+
+	/**
+	* Defines the content in the middle column.
+	* @type {HTMLElement}
+	* @slot
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.midColumn
+	* @public
+	*/
+	@slot()
+	midColumn!: Array<HTMLElement>;
+
+	/**
+	* Defines the content in the end column.
+	* @type {HTMLElement}
+	* @slot
+	* @name sap.ui.webc.fiori.FlexibleColumnLayout.prototype.endColumn
+	* @public
+	*/
+	@slot()
+	endColumn!: Array<HTMLElement>;
+
+	initialRendering: boolean;
+	_handleResize: () => void;
+	static i18nBundle: I18nBundle;
+	_prevLayout: FCLLayout | null;
+
+	constructor() {
+		super();
+
+		this._prevLayout = null;
+		this.initialRendering = true;
+		this._handleResize = this.handleResize.bind(this);
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return FlexibleColumnLayoutCss;
+	}
+
+	static get template() {
+		return FlexibleColumnLayoutTemplate;
+	}
+
+	static get dependencies() {
+		return [Button];
+	}
+
+	static async onDefine() {
+		FlexibleColumnLayout.i18nBundle = await getI18nBundle("@ui5/webcomponents-fiori");
+	}
+
+	static get ANIMATION_DURATION() {
+		return getAnimationMode() !== AnimationMode.None ? 560 : 0;
+	}
+
+	onEnterDOM() {
+		ResizeHandler.register(this, this._handleResize.bind(this));
+	}
+
+	onExitDOM = () => {
```

Comment:
Is there a reason to change 
"onExitDOM() {
to
onExitDOM = () => {

Reply:
I changed it since, it was necessary in order for `lint` to be successful after for the first [PR](https://github.com/SAP/ui5-webcomponents/pull/6309). However I see that now, after the code review changes, it is not necessary anymore, so I am reverting it to the old way.